src/main/java/io/stargate/sgv2/jsonapi/config/LwtConfig.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.sgv2.jsonapi.config;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.PositiveOrZero;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Configuration setup for the Light-weigh transactions.
+ */
+@ConfigMapping(prefix = "stargate.jsonapi.lwt")
+public interface LwtConfig {
+
+    /** @return Defines the maximum retry for lwt failure <code>3</code>. */
+    @Max(5)
+    @PositiveOrZero
```

Comment:
@maheshrajamani changed `@Positive` to `@PositiveOrZero` so one can disable retires, does this make sense?

Reply:
@ivansenic I think that makes sense, +1.

Reply:
@maheshrajamani what do you mean it's coded to retry at least once on LWT failure? What does this mean? Wouldn't setting this to `0` allow anybody to say don't retry on the LWT failure? Would it not work?

Reply:
Yes. This it will retry at least once in case of LWT.

Reply:
Done.